### PR TITLE
Add remove Wi-Fi network GATT characteristic

### PR DESCRIPTION
**Pull request by fvasquez**
Tested with nRF Connect by writing "Bingo" to new GATT characteristic.

```
# tail -f /var/data/log/gateway_config/console.log 
2020-03-25 01:57:19.975 [warning] <0.533.0>@gateway_config_worker:init_ubx:99 No UBX filename or device found, running in stub mode
2020-03-25 01:57:20.001 [info] <0.544.0>@gateway_gatt_char_wifi_ssid:update_value:83 WiFi still disconnected
2020-03-25 01:58:21.032 [info] <0.533.0>@gateway_config_worker:handle_info:218 Button clicked
2020-03-25 01:58:21.036 [info] <0.533.0>@gateway_config_worker:handle_info:226 Enabling advertising
2020-03-25 01:58:22.067 [info] <0.554.0>@ble_advertisement:handle_info:99 Started advertisement gateway_ble_advertisement
2020-03-25 01:58:57.174 [info] <0.544.0>@gateway_gatt_service:handle_info:66 Trying to connect to WiFI SSID: "RT-AC66U_B1_38_2G"
2020-03-25 01:58:57.177 [info] <0.560.0>@connman_connect:connecting:62 Connecting to "RT-AC66U_B1_38_2G" at "/net/connman/service/wifi_6081f9042e73_52542d41433636555f42315f33385f3247_managed_psk"
2020-03-25 01:58:57.180 [info] <0.559.0>@connman_agent:handle_message:40 Responded to input request for path "/net/connman/service/wifi_6081f9042e73_52542d41433636555f42315f33385f3247_managed_psk"
2020-03-25 01:59:03.212 [info] <0.544.0>@gateway_gatt_service:handle_info:85 Connect result ok
2020-03-25 01:59:03.249 [info] <0.544.0>@gateway_gatt_char_wifi_ssid:update_value:87 WiFi connected to: "RT-AC66U_B1_38_2G" at "/net/connman/service/wifi_6081f9042e73_52542d41433636555f42315f33385f3247_managed_psk"
2020-03-25 02:56:40.082 [warning] <0.544.0>@gateway_gatt_char_wifi_remove:write_value:35 Failed to parse wifi_remove request: {gpb_error,{decoding_failure,{<<"Bingo">>,gateway_wifi_remove_v1_pb,{error,{badmatch,<<"ngo">>},[{gateway_gatt_char_wifi_remove_pb,skip_length_delimited_gateway_wifi_remove_v1_pb,5,[{file,"gateway_gatt_char_wifi_remove_pb.erl"},{line,344}]},{gateway_gatt_char_wifi_remove_pb,decode_msg_1_catch,3,[{file,"gateway_gatt_char_wifi_remove_pb.erl"},{line,226}]},{gateway_gatt_char_wifi_remove,write_value,2,[{file,"gateway_gatt_char_wifi_remove.erl"},{line,30}]},{gatt_characteristic,handle_message_characteristic,3,[{file,"gatt_characteristic.erl"},{line,190}]},{gatt_service,handle_message_characteristic,5,[{file,"gatt_service.erl"},{line,250}]},{ebus_object,handle_info,2,[{file,"ebus_object.erl"},{line,122}]},{gen_server,try_dispatch,4,[{file,"gen_server.erl"},{line,637}]},{gen_server,handle_msg,6,[{file,"gen_server.erl"},{line,711}]}]}}}}
```

Also wrap `Why` reasons from gpb `decode_msg` exceptions in square brackets so stacktraces get logged.

### CODE REVIEW

File: include/gateway_config.hrl [hunk 1]
```diff
@@ -16,7 +16,6 @@
 -define(MINER_MEMBER_ADD_GW, "AddGateway").
 -define(MINER_MEMBER_ASSERT_LOC, "AssertLocation").
 -define(MINER_MEMBER_P2P_STATUS, "P2PStatus").
--define(MINER_MEMBER_ETH_ONLINE, "EthernetOnline").
```
Comment (fvasquez) [inline include/gateway_config.hrl]: Not referenced anywhere.

Comment (madninja) [reply include/gateway_config.hrl]: Right, we're not asking miner about ethernet, good catch

File: rebar.config [hunk 2]
```diff
@@ -40,7 +41,7 @@
         {h3, ".*", {git, "https://github.com/helium/erlang-h3.git", {branch, "master"}}},
         {ubx, ".*", {git, "https://github.com/helium/erlang-ubx", {branch, "master"}}},
         {gatt, ".*", {git, "https://github.com/helium/ebus-gatt", {branch, "master"}}},
-        {connman, ".*", {git, "https://github.com/helium/ebus-connman", {branch, "master"}}},
+        {connman, ".*", {git, "https://github.com/helium/ebus-connman", {branch, "remove-wifi-service"}}},
```
Comment (madninja) [inline rebar.config]: We need ebus-connman to merge so we can fix this up, right?

Comment (madninja) [reply rebar.config]: That ebus-connman  feature is now merger: https://github.com/helium/ebus-connman/pull/4 Thanks @fvasquez

**Comment by fvasquez** [comment]
I plan to revisit this GATT characteristic and make its value readable so that I can signal a result of success or failure back to the mobile app. Removing an SSID is an async operation that behaves a lot like connect does when a hotspot is connected to the SSID you are removing.

**Comment by madninja** [comment]
Yeah, I'm pretty are sure the mobile app will need some response back for better UX

**Pull request merged**